src/libslic3r/Point.hpp

```diff
@@ -116,6 +116,8 @@ class Point : public Vec2crd
 
     Point& operator+=(const Point& rhs) { (*this)(0) += rhs(0); (*this)(1) += rhs(1); return *this; }
     Point& operator-=(const Point& rhs) { (*this)(0) -= rhs(0); (*this)(1) -= rhs(1); return *this; }
+	Point& operator/=(const int &rhs) { (*this)(0) = coord_t((double)(*this)(0) / (double)rhs); (*this)(1) = coord_t((double)(*this)(1) / (double)rhs); return *this; }
```

Comment:
I would prefer to not extend our homebrew Point type and rather use the functionality provided by the Eigen library.